haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
```

Comment:
The inclusion of the word "initial" was quite purposeful there. You're right that it's "just" good practices, but the interface should encourage that. For consistency I can change it to `create_initial_admin`. If you feel strongly I can get rid of the "initial" bit, but I think it's a good idea to have it in there.